website/content/blog/2023-11-getting-started-as-a-tag-env-contributor.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Getting started as a TAG Environmental Sustainability Contributor
+description: Getting started as a TAG Environmental Sustainability Contributor
+slug: tbd
+weight: 1
+date:   2023-11-07 00:00:00 +0000
+author: Antonio Di Turi, Atreay Kukanur, Mfon Nta
+reviewers: Kristina Devochko, Leonard Pahlke, Marlow Weston
+---
+
+## Getting started as a TAG Environmental Sustainability Contributor
+
+<p class="mt-5 mb-5"><img src="/images/blogs/2023-11-getting-started-as-a-tag-environmental-sustainability-contributor/tag-env-logo.png" alt="TAG Environmental Sustainability Logo"></p>
```

Comment:
we need to update the image references to load the pictures

`2023-11-getting-started-as-a-tag-env-contributor`

Reply:
solved!